client/app.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect} from 'react';
+
+import { hot } from 'react-hot-loader/root';
+
+import { postPizza } from './asyncActions'
+
+const App = () => {
+  useEffect(() => {
+      postPizza()
```

Comment:
@AStashevskaya Lets avoid `seeding` database date from client app, BTW it throws error:

<img width="522" alt="image" src="https://user-images.githubusercontent.com/12561302/113560405-3d278300-9603-11eb-9bfe-4d03bb98138b.png">

Lets create script called `seed` and let user to manually run it with `node` instead